appveyor.yml

```diff
@@ -1,9 +1,5 @@
 version: "{build}"
 
-branches:
-  only:
-    - master
```

Comment:
@50Wliu there may be... this was making it so that the *-releases branches weren't building. Maybe we should do `master` *and* a regex pattern for the releases branches. Do you want to open a tiny PR to do so? That's probably what I should have done but my brain was mush from waiting on builds all day.